app/views/users/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_with model: @user do |form| %>
+    <div class="form-group mb-3">
+      <%= form.label :username, class: "form-label"  %>
+      <%= form.text_field :username, class: "form-control" %>
+    </div>
+
+    <div class="form-group mb-3">
+      <%= form.label :email, class: "form-label"  %>
+      <%= form.text_field :email, class: "form-control" %>
+    </div>
+
+    <div class="form-group mb-3">
+      <%= form.label :password, class: "form-label"  %>
+      <%= form.text_field :password, type: 'password', class: "form-control" %>
+    </div>
+
+    <div class="form-group d-grid">
+      <%= form.submit nil, class: 'btn btn-success btn-sm' %>
+    </div>
+  <% end %>
```

Comment:
1. In line 9 instead of `text_field` use `email_field`
2. In line 14 instead of `text_field` use `password_field`